client/lib/post-normalizer/rule-content-make-images-safe.js

```diff
@@ -60,7 +60,7 @@ function isCandidateForContentImage( imageUrl ) {
 	} );
 }
 
-export default function( maxWidth ) {
```

Comment:
@blowery, can you explain a little more how this broke things?
I'm still unsure if this bug is affecting `posts-lists` (the only other client of this functionality).